srcpkgs/rocksdb/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'rocksdb'
+pkgname=rocksdb
+version=5.17.2
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release -DWITH_TESTS=0 -DUSE_RTTI=0 -DWITH_LZ4=1"
+hostmakedepends="git"
+makedepends="liblz4-devel"
+depends="liblz4"
```

Comment:
Shouldn't be necessary if it links to this library, just need the -devel in makedepends